js/tests/unit/modal.spec.js

```diff
@@ -11,8 +11,8 @@ describe('Modal', () => {
   beforeAll(() => {
     fixtureEl = getFixture()
 
-    // Enable the scrollbar measurer
-    const css = '.modal-scrollbar-measure { position: absolute; top: -9999px; width: 50px; height: 50px; overflow: scroll; }'
+    // Enable the scrollbar measurer and reset body margin
```

Comment:
@kremerd my only concern is that this should be set only for the newly added test. What do you think?

Reply:
@XhmikosR Yes, you're right. I originally thought it might make sense for all scrollbar related tests, but that is neither necessary, nor within the scope of this PR. I pushed a patch to restrict the setting.

Since I'm still new to the PR process: Was it right to publish this as a new commit, or should I rather have amended my previous one and force pushed?

Reply:
@kremerd it's OK, ideally should have been squashed, but I squashed it myself. Thanks!